Signal/src/view controllers/CodeVerificationViewController.m

```diff
@@ -139,6 +139,7 @@ - (void)createViews {
                                                         @"Text field placeholder for SMS verification code during registration");
     _challengeTextField.font = [UIFont ows_lightFontWithSize:21.f];
     _challengeTextField.textAlignment = NSTextAlignmentCenter;
+    _challengeTextField.keyboardType = UIKeyboardTypePhonePad;
```

Comment:
> the "code verification" view used the conventional keyboard. 

Yikes, that's a regression. Previously you'd see a numeric only keypad which omits the special non-numeric phone characters (+X#), which seems preferable. 

![digit-pad](https://cloud.githubusercontent.com/assets/217057/23073922/22697bc0-f505-11e6-9f81-b78e7c4ae31f.jpg)

This was on a physical 5c running 2.7.1